src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdk.java

```diff
@@ -357,12 +363,7 @@ public CloudSdk build() {
 
       // Default SDK path
       if (sdkPath == null) {
-        Path discoveredSdkPath = new PathResolver().getCloudSdkPath();
-        if (discoveredSdkPath == null) {
-          throw new AppEngineException("Google Cloud SDK path was not provided and could not be"
-              + " found in any known install locations.");
-        }
-        sdkPath = discoveredSdkPath;
+        sdkPath = discoverSdkPath();
```

Comment:
To see the difference it makes in the Eclipse tools: https://github.com/GoogleCloudPlatform/gcloud-eclipse-tools/pull/362/files